### DISCUSSION TITLE
Space Theme on VSCode ?

### DISCUSSION THREAD

**Discussion by Kae-Tempest** [aliakbarmostafaei/jetbrains-space-theme#25] [Polls]
Can you make Space Theme on VScode ?

**Comment by aliakbarmostafaei** [comment]
I have it in my plans, right now I'm taking rest from health issues which might take a few weeks, thank you for your interest and support.

**Reply by Kae-Tempest** [reply]
Hello, are you good ? No update since a while. take care of you man ( or girl ? )
